specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -210,167 +197,26 @@ function TreeView<SCHEMA extends AnyTree>({
-          pt-0
-          shadow-md shadow-gray-500 outline-none
-          ${highContrast ? 'border dark:border-white' : 'bg-gradient-to-bl'}
-        `}
-        role="none table"
-        // First role is for screen readers. Second is for styling
-        style={
-          {
-            '--cols': treeDefinitionItems.length,
-            '--middle-color': `${treeAccentColor}33`,
-            '--edge-color': `${treeAccentColor}00`,
-          } as React.CSSProperties
-        }
-        tabIndex={0}
-        // When tree viewer is focused, move focus to last focused node
-        onFocus={(event): void => {
-          // Don't handle bubbled events
-          if (event.currentTarget !== event.target) return;
-          // If user wants to edit tree ranks, allow tree ranks to receive focus
-          if (isEditingRanks) return;
-          event.preventDefault();
-          // Unset and set focus path to trigger a useEffect hook in <TreeNode>
-          setFocusPath([-1]);
-          globalThis.setTimeout(
-            () => setFocusPath(focusPath.length > 0 ? focusPath : [0]),
-            0
-          );
-        }}
-      >
-        <div role="none rowgroup">
-          <div role="none row">
-            {treeDefinitionItems.map((rank, index, { length }) => {
-              const rankName = rank.title || rank.name;
-              return (
-                <div
-                  className={`
-                  sticky top-0 whitespace-nowrap border border-transparent p-2
-                  ${index === 0 ? '-ml-2 rounded-bl pl-4' : ''}
-                  ${index + 1 === length ? '-mr-2 rounded-br pr-4' : ''}
-                  ${
-                    reduceTransparency || !supportsBackdropBlur
-                      ? 'bg-gray-100 dark:bg-neutral-900'
-                      : 'bg-gray-100/60 backdrop-blur-sm dark:bg-neutral-900/60'
-                  }
-                `}
-                  key={index}
-                  role="columnheader"
-                >
-                  <Button.LikeLink
-                    id={id(rank.rankId.toString())}
-                    onClick={(): void =>
-                      setCollapsedRanks(
-                        toggleItem(collapsedRanks ?? [], rank.rankId)
-                      )
-                    }
-                  >
-                    {
-                      (collapsedRanks?.includes(rank.rankId) ?? false
-                        ? rankName[0]
-                        : rankName) as LocalizedString
-                    }
-                  </Button.LikeLink>
-                  {isEditingRanks &&
-                  collapsedRanks?.includes(rank.rankId) !== true ? (
-                    <EditTreeRank rank={rank} />
-                  ) : undefined}
-                </div>
-              );
-            })}
-          </div>
+      {isSplit ? (
+        <div className="h-full w-full">
+          <Splitter
+            key="test"
+            position={'horizontal'}
```

Comment:
hmm, I though you were implementing a vertical split, not horizontal.
horizontal split doesn't much sense, unless you have a square monitor (on specify open in half screen)
trees when expanded are very tall, not very wide

@grantfitzsimmons thougs?

Reply:
Can we ask the #sp7-7-ui-ux channel about this? It's interesting– I figured we would recreate the 6 tree split interface but it was conceived in a different era.

Reply:
ah, I didn't remember what sp6 looked like, but just assumed that vertical split makes more sense given the common dimensions of monitors, and the shape of the tree

(i.e having windows side by side is common, or having one or two sidebars in a webpage is common)

also ask ChatGPT

but feel free to ignore me: I am not a biologist, and not the one who will be using this feature

Reply:
edit:
the videos Caroline sent (https://github.com/specify/specify7/pull/3969#issuecomment-1718282639) indeed work better with horizontal - but those videos contain test data where a state has only a few counties

in real trees, genus often contains a lot of species, or subspecies contains many species, and (I am assuming) it's more valuable to be able to see neighboring species within a given genus, than neighboring genera - thus the data is taller rather than wider